content/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists.md

```diff
@@ -12,24 +12,23 @@ versions:
   ghae: '*'
   ghec: '*'
 ---
+
 ## Forking gists
 
 Each gist indicates which forks have activity, making it easy to find interesting changes from others.
 
-![Gist forks](/assets/images/help/gist/gist_forks.png)
-
 ## Cloning gists
 
 If you want to make local changes to a gist and push them up to the web, you can clone a gist and make commits the same as you would with any Git repository. For more information, see "[AUTOTITLE](/repositories/creating-and-managing-repositories/cloning-a-repository)."
 
-![Gist clone button](/assets/images/help/gist/gist_clone_btn.png)
+To clone a gist, select the Embed dropdown menu, then click Clone via HTTPS or Clone via SSH.
+
+![Screenshot of the Embed dropdown from GitHub Gist. The Embed dropdown is expanded and an option labeled “Clone via HTTPS” is outlined in dark orange.](/assets/images/help/gist/gist_clone_btn.png)
 
 ## Viewing gist commit history
 
 To view a gist's full commit history, click the "Revisions" tab at the top of the gist.
 
-![Gist revisions tab](/assets/images/help/gist/gist_revisions_tab.png)
-
 You will see a full commit history for the gist with diffs.
 
-![Gist revisions page](/assets/images/help/gist/gist_history.png)
+![Screenshot of the GitHub Gist Revisions page. A tab labeled “Revisions” with dark orange underline is outlined in dark orange.](/assets/images/help/gist/gist_history.png)
```

Comment:
I'm not sure the dark orange underline is important to mention, and it just makes the sentence a bit hard to parse with the second dark orange later. 😊 
```suggestion
![Screenshot of the "Revisions" page in GitHub Gist. A tab, labeled “Revisions”, is outlined in dark orange.](/assets/images/help/gist/gist_history.png)
```